DogeNet.DAL/Models/User.cs

```diff
@@ -6,8 +6,11 @@ namespace DogeNet.DAL.Models
 {
     using System;
     using System.Collections.Generic;
+    using System.Security.Claims;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Identity;
 
-    public partial class User
+    public partial class User : IdentityUser<int>
```

Comment:
Need to find a way do not to inherit classes from identity classes

Reply:
Why?

Reply:
We have our User from database and required user from Identity, we can't create 2 tables, can we?

Reply:
I think it would be better to have two separate tables: one related to our business needs and other for Identity Server